app/services/early_career_teachers/create.rb

```diff
@@ -15,7 +15,9 @@ def call
           profile.school = school_cohort.school
         end
 
-        ParticipantProfile::ECT.create!({ teacher_profile: teacher_profile, schedule: Finance::Schedule.default }.merge(ect_attributes))
+        ParticipantProfile::ECT.create!({ teacher_profile: teacher_profile, schedule: Finance::Schedule.default }.merge(ect_attributes)) do |profile|
+          ParticipantProfileState.create!({ participant_profile: profile })
+        end
```

Comment:
We don't need to populate it. If it's nil, it's active. I did think about not populating it at all until an action is performed on it. That's another option.

Reply:
> If it's nil, it's active

Ok, then should we not populate it here either? Otherwise some teachers will have an active state object in the database, and others won't.